.travis.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       os: linux
       script: 
         - yes | conda install -c conda-forge fsleyes=0.33.1
+        - yes | conda install -c conda-forge h5py=2.10.0
```

Comment:
why this change? pls explain in your PR-- seems out of scope when not justified

Reply:
Yup  - if necessary, it needs to be done in a new PR please.

Reply:
This is an older commit from PR #392